values.schema.json

```diff
@@ -1121,12 +1121,11 @@
                                     "minimum": 1,
                                     "description": "64GiB is recommended for good performance"
                                 },
-                                "slackGiB": {
```

Comment:
Maybe we should leave this slackGIB for the disk columns?

Reply:
I remember Vincent having some troubles when the PV size was exactly the same as the configured disk size, hence the slack. Either the truly allocated size was a bit smaller or ndb is allocating a bit more.